cmd/machine-controller/main.go

```diff
@@ -156,6 +158,7 @@ func main() {
 	flag.StringVar(&nodeKubeletRepository, "node-kubelet-repository", "quay.io/kubermatic/kubelet", "Repository for the kubelet container. Only has effect on Flatcar Linux, and for kubernetes >= 1.18.")
 	flag.StringVar(&nodeKubeletFeatureGates, "node-kubelet-feature-gates", "RotateKubeletServerCertificate=true", "Feature gates to set on the kubelet. Default: RotateKubeletServerCertificate=true")
 	flag.StringVar(&nodeContainerRuntime, "node-container-runtime", "docker", "container-runtime to deploy")
+	flag.StringVar(&caBundleFile, "ca-bundle", "", "(optional) path to a file containing all PEM-encoded CA certificates (will be used instead of the host's certificates)")
```

Comment:
Nit: Just to follow the pattern as the previous flags(I don't think the optional hint)
```suggestion
	flag.StringVar(&caBundleFile, "ca-bundle", "", "Path to a file containing all PEM-encoded CA certificates (will be used instead of the host's certificates if set)")
```